src/ast.rs

```diff
@@ -4735,6 +4735,7 @@ impl Parse for Expression {
                 b"cl" => {
                     let (func, tail) = Expression::parse(ctx, subs, tail)?;
                     let (args, tail) = zero_or_more::<Expression>(ctx, subs, tail)?;
+                    let tail = consume(b"E", tail)?;
```

Comment:
Good catch! The grammar also lists this as

```
<expression> ::= cl <expression>+ E
```

too, so this must have jsut been an oversight.

http://itanium-cxx-abi.github.io/cxx-abi/abi.html#mangle.expression